include/aspect/simulator.h

```diff
@@ -1391,6 +1390,8 @@ namespace aspect
 
       DoFHandler<dim>                                           dof_handler;
 
+      Postprocess::Manager<dim>                                 postprocess_manager;
+
```

Comment:
any particular reason to move the place where we declare this manager (rather than sticking with all of the other managers)? is it because you need to have access to the `dof_handler` when you initialize it?